tensorflow/python/saved_model/README.md

```diff
@@ -76,7 +76,56 @@ saved_model.pb
 ## APIs
 The APIs for building and loading a SavedModel are described in this section.
 
+### Simple
+
+The highest level API for exporting the model components described above
+is using the [simple_save](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/saved_model/simple_save.py)
```

Comment:
Use `@{tf.saved_model.simple_save}` and load everywhere. For top-level info, use `@{$guide/saved_model}`.